remo/events/tests/test_api.py

```diff
@@ -67,3 +70,91 @@ def test_get_remo_url(self):
         data = EventDetailedSerializer(event,
                                        context={'request': request}).data
         ok_(event.get_absolute_url() in data['remo_url'])
+
+
+class TestEventsKPIView(RemoTestCase):
+    def setUp(self):
+        self.factory = RequestFactory()
+        self.url = '/api/kpi/events'
+
+    def test_total(self):
+        EventFactory.create_batch(10)
+        request = self.factory.get(self.url)
+        request.QUERY_PARAMS = dict()
+        response = EventsKPIView().get(request)
+        eq_(response.data['total'], 10)
+
+    @patch('remo.events.api.views.now')
+    def test_quarter(self, mock_now):
+        mock_now.return_value = datetime(2015, 3, 1)
+
+        # Previous quarter
+        start = datetime(2014, 12, 5)
+        end = datetime(2014, 12, 6)
+        EventFactory.create_batch(10, start=start, end=end)
```

Comment:
There's no need to create so many events. You can make your test case with only one event per quarter for example.